samples/web/content/testrtc/js/camresolutionstest.js

```diff
@@ -91,11 +91,12 @@ CamResolutionsTest.prototype = {
     var call = new Call();
     call.pc1.addStream(stream);
     call.establishConnection();
-    call.gatherStats(call.pc1, this.analyzeStats_.bind(this), 1000);
-    setTimeoutWithProgressBar( function() {
+    this.collectStatsStartTime = Date.now();
```

Comment:
Not sure if this is good enough, should I leave it or move it to call.gatherStats?